stdlib/private/SwiftPrivateLibcExtras/SwiftPrivateLibcExtras.swift

```diff
@@ -37,36 +37,36 @@ public var _stdlib_FD_SETSIZE: CInt {
   return 1024
 }
 
-public struct _stdlib_fd_set {
-  var _data: [UInt32]
+public struct __stdlib_fd_set<T: UnsignedInteger> {
```

Comment:
The original implementation aliases an array of C long int's with an array of Swift UInt32's, regardless of architecture word size, which happened to work fine for little-endian systems but not for big-endian ones.